stream-compositions/services/payment-order-composition-service/src/main/resources/banner.txt

```diff
@@ -0,0 +1,8 @@
+,------.                                                   ,--.      ,-----.             ,--.                    ,-----.                                            ,--.   ,--.   ,--.
+|  .--. '  ,--,--. ,--. ,--. ,--,--,--.  ,---.  ,--,--,  ,-'  '-.   '  .-.  ' ,--.--.  ,-|  |  ,---.  ,--.--.   '  .--./  ,---.  ,--,--,--.  ,---.   ,---.   ,---.  `--' ,-'  '-. `--'  ,---.  ,--,--,
+|  '--' | ' ,-.  |  \  '  /  |        | | .-. : |      \ '-.  .-'   |  | |  | |  .--' ' .-. | | .-. : |  .--'   |  |     | .-. | |        | | .-. | | .-. | (  .-'  ,--. '-.  .-' ,--. | .-. | |      \
+|  | --'  \ '-'  |   \   '   |  |  |  | \   --. |  ||  |   |  |     '  '-'  ' |  |    \ `-' | \   --. |  |      '  '--'\ ' '-' ' |  |  |  | | '-' ' ' '-' ' .-'  `) |  |   |  |   |  | ' '-' ' |  ||  |
+`--'       `--`--' .-'  /    `--`--`--'  `----' `--''--'   `--'      `-----'  `--'     `---'   `----' `--'       `-----'  `---'  `--`--`--' |  |-'   `---'  `----'  `--'   `--'   `--'  `---'  `--''--'
+                   `---'                                                                                                                    `--'
```

Comment:
For other services we used "straight" banner font, but if we want to change it, let do it for other compositions as well, for consistency.

Reply:
In RnD we have the (ignored) guideline to not do any banner. Customers will disable in production.

Reply:
I have removed the banner. Going to mark this as resolved, let me know if I need to do this otherwise.